doc/guide/packages.xml

```diff
@@ -310,6 +310,42 @@ mypackage/test.min.js.gz
       </varlistentry>
     </variablelist>
 
+    <para>The <code>spawn</code> and <code>environ</code> values can be dynamically
+    taken from a matching open command values. When a value in either the <code>spawn</code>
+    or <code>environ</code> array contains a named variable wrapped in <code>${}</code>,
+    the variable will be replaced with the value contained in the matching open command.
+    Only named variables are supported and name can only contain letters, numbers and
+    the following symbols: <code>._-</code></para>
+
+    <para>If you need to include <code>${}</code>, as an actual value in your arguments
+    you can escape it by prefixing it with a <code>\</code></para>
+
+    <para>For example a bridges section like:</para>
+<programlisting>
+{
+  "bridges": [
+    {
+      "match": { "payload": "example" },
+      "environ": [ "TAG=${tag}" ],
+      "spawn: [ "/example-bridge", "--tag", "${tag}" "\${other}"],
+      "problem": "access-denied"
+    }
+  ]
+}
+</programlisting>
+
+    <para>when a open command is received with a payload of <code>example</code>
+        with <code>tag</code> value of <code>tag1</code>. The following
+        command will be spawned</para>
+
+    <programlisting>TAG=tag1 /example-bridge --tag tag1 ${other}</programlisting>
```

Comment:
This example makes it look like the shell is used to spawn the commands. It's not the shell. Perhaps the inclusion of an escaped argument is not necessary?